src/libraries/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/AesImplementation.Browser.cs

```diff
@@ -0,0 +1,27 @@
+    // Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal partial class AesImplementation
+    {
+        private static ICryptoTransform CreateTransformCore(
+            CipherMode cipherMode,
+            PaddingMode paddingMode,
+            byte[] key,
+            byte[]? iv,
+            int blockSize,
+            bool encrypting)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        // -----------------------------
```

Comment:
@bartonjs do you know what these comments indicate? I looked at some of the 51 files and it isn't clear to me what they are separating.

Reply:
Nope, just an artifact from the early porting/x-plat days.